app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years,expected_result",
+    [
+        pytest.param(
+            14, 14, [0, 0],
+            id="return zeros if animal's years less than 15"
+        ),
+        pytest.param(
+            15, 15, [1, 1],
+            id="return 1 human's year if animal's years equal to 15"
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="return 1 human's year if animal's years less than 24"
```

Comment:
The description 'return 1 human's year if animal's years less than 24' might be misleading. According to the task description, 23 years should still return 1 human year, which is correct, but the phrasing could imply that 23 is not included. Consider rephrasing for clarity.